src/Event/AfterPersist.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the zenstruck/foundry package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Foundry\Event;
```

Comment:
```suggestion
namespace Zenstruck\Foundry\Persistence\Event;
```

?

Reply:
I moved other ones to `Zenstruck\Foundry\Object\Event` as well